### PR TITLE
Implemented server classes

### PR DESCRIPTION
**Pull request by ccapiferri**

### CODE REVIEW

**Comment by adellafrattina** [comment]
javadoc missing 👎

**Comment by ccapiferri** [comment]
😭

**Comment by adellafrattina** [comment]
Still no javadoc? 😞 👎

**Comment by ccapiferri** [comment]
0dc9027cde1bf33954d95f62960451f321bc0e340ddb28c049a7a173c4a372e9_1-2826787729

File: climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java [hunk 1]
```diff
@@ -1,6 +1,346 @@
...
+
+			catch (ClassNotFoundException ex) {
+
+				Console.write("Class of a serialized object cannot be found");
+				m_running = false;
+			}
+		}
+
+		close();
+	}
```
Comment (adellafrattina) [inline climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java]: At line 319, you should add a console message to print that a client has disconnected

**Review by adellafrattina — COMMENTED** [review]
This looks good, only 2 notes: ServerDatabaseImpl.java methods need to be synchronized + watch down below

**Comment by adellafrattina** [comment]
Now you need to pull the new updated main branch to change the DatabaseRequestException handling where needed

**Comment by adellafrattina** [comment]
Hurry up lil bro

File: climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java (hunk 1 above)
Comment (ccapiferri) [reply climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java]: Fixed

**Review by adellafrattina — APPROVED** [review]
LGTM!

**Pull request merged**